Makefile

```diff
@@ -30,6 +31,11 @@ tools: check-env toitpkg toitlsp build/host/bin/toitvm build/host/bin/toitc
 .PHONY: esp32
 esp32: check-env build/esp32/toit.bin
 
+.PHONY: flash
+flash: esp32
```

Comment:
Maybe depend on check-env too and move any printing of IDF_PATH there?

Reply:
Maybe it would be cleaner to depend on build/esp32/toit.bin? We could expand that to all the .bin files and make it clear that they are produced by the same rule.